approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "scienzecognitive"
```

Comment:
questa la toglierei, è fuori contesto pure in Lobste.rs e non la usa nessuno

Reply:
Ok, la tolgo.